test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -372,6 +379,61 @@ function primitive_to_prognostic!(
 
 end
 
+function compute_filter_argument!(
+    target::IntrinsicEDMFFilter,
```

Comment:
I think the notation we use throughout the model is "specific" for intensive variables per unit mass. See #2116 . Intrinsic does not have a well established meaning in this context.